src/Event.tsx

```diff
@@ -60,14 +77,14 @@ class Event extends React.Component<EventProps, EventState> {
             <YAMLViewer object={event} />
           </Tab>
           {event.git ? (
-            <Tab eventKey="git-initializer-logs" title="Git Initializer Logs">
+            <Tab eventKey="git-initializer-logs" title="Git Initializer Logs" disabled = {this.state.grayedOut}>
               <LogStreamer event={event} containerName="vcs" logKey="vcs" />
             </Tab>
           ) : null}
-          <Tab eventKey="worker-logs" title="Worker Logs">
+          <Tab eventKey="worker-logs" title="Worker Logs" disabled = {this.state.grayedOut}>
             <LogStreamer event={event} logKey={event?.metadata?.id || ""} />
           </Tab>
-          <Tab eventKey="jobs" title="Jobs">
+          <Tab eventKey="jobs" title="Jobs" disabled = {this.state.grayedOut}>
```

Comment:
This shouldn't be grayed out if the user doesn't have access to the logs.

It is the log tabs _on_ each job that should be grayed out.

Reply:
Done with it. Will be there in the next commit.